src/test/java/UiTest.java

```diff
@@ -0,0 +1,33 @@
+import karen.Ui;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UiTest {
+    private final PrintStream standardOut = System.out;
```

Comment:
Shouldn't constant names be all uppercase using underscore to separate words?

Reply:
I didn't realised I overlooked the checkstyle checks for my test cases until you brought it up, thanks!